edb/edgeql/pygments/__init__.py

```diff
@@ -25,8 +25,13 @@
 
 __all__ = ['EdgeQLLexer']
 
+# some unreserved keywords have special handling because they clash
+# with builtins
+EXCEPTION_KW = {'named'}
```

Comment:
Don't hardcode this; names like this should be available via `meta.py`

Reply:
I'm not sure yet how to codify it properly. The basic idea is that unreserved keywords like this will generally have some special context, like another keyword before or after, or potentially some other symbol. Well, technically, *every* unreserved keyword has some special context, hence "unreserved", but I don't yet know what's the best way to capture it.
Maybe I could do some sort of tuple like `(before, kw, after)`, so for named it would be `(before=None, kw='named', after='only')`.

Reply:
The bottom line is that there's a more general problem that needs to be solved - how to handle unreserved keywords. That can be addressed in a separate PR, but for the moment the hardcode at least indicates that something special is being done to what otherwise would be a keyword.